src/Symfony/Component/Validator/Tests/Constraints/LessThanValidatorWithNegativeConstraintTest.php

```diff
@@ -65,19 +65,19 @@ public function testThrowsConstraintExceptionIfValue()
     /**
      * @dataProvider provideInvalidConstraintOptions
      */
-    public function testThrowsConstraintExceptionIfNoValueOrPropertyPath($options)
-    {
-        $this->expectException(ConstraintDefinitionException::class);
-        $this->expectExceptionMessage('requires either the "value" or "propertyPath" option to be set.');
-        $this->markTestSkipped('Value option always set for Negative constraint');
-    }
-
-    public function testThrowsConstraintExceptionIfBothValueAndPropertyPath()
-    {
-        $this->expectException(ConstraintDefinitionException::class);
-        $this->expectExceptionMessage('requires only one of the "value" or "propertyPath" options to be set, not both.');
-        $this->markTestSkipped('Value option is set for Negative constraint automatically');
-    }
+//    public function testThrowsConstraintExceptionIfNoValueOrPropertyPath($options)
+//    {
+//        $this->expectException(ConstraintDefinitionException::class);
+//        $this->expectExceptionMessage('requires either the "value" or "propertyPath" option to be set.');
+//        $this->markTestSkipped('Value option always set for Negative constraint');
+//    }
+//
+//    public function testThrowsConstraintExceptionIfBothValueAndPropertyPath()
+//    {
+//        $this->expectException(ConstraintDefinitionException::class);
+//        $this->expectExceptionMessage('requires only one of the "value" or "propertyPath" options to be set, not both.');
+//        $this->markTestSkipped('Value option is set for Negative constraint automatically');
+//    }
```

Comment:
Can someone explain these tests to me? we expect an exception and mark the test as skipped? 🤔 I think the expectedException*() calls can be removed?

Reply:
I am wondering how PHPUnit behaves if an exception is expected, but no exception is thrown? Shouldn't this fail?

Reply:
Ok I marked the tests skipped, like the other ones down below.